app/_hub/kong-inc/aws-lambda/index.md

```diff
@@ -240,18 +255,19 @@ from being executed. We are planning to remove this limitation in the future.
 
     Test the lambda function from the AWS console and make sure the execution succeeds.
 
-4. Finally we set up a Service and Route in Kong and link it to the function just created.
+4. Finally, we set up a Service and Route in Kong and link it to the
+`Mylambda` function we just created.
 
-{% tabs %}
-{% tab With a database %}
+{% navtabs %}
+{% navtab With a database %}
```

Comment:
Currently everything in these tabs appears outside of the steps - is this intended behaviour? I'd nest this in a step and then continue with "After everything is created, call the Service..." as the final step.

Reply:
I'm not sure what the original author's intent was. I was just getting the tabs to work. 
![image](https://user-images.githubusercontent.com/3756245/82226464-23e2ee80-98ec-11ea-89cd-d51899b28684.png)
I'll try nesting in the step.

Reply:
@lena-larionova I attempted to indent and make a numbered step but was unsuccessful. 

It makes the tabs not work even though I tired carefully lining them up. Also it breaks the numbering sequence and restarts at 1.

![image](https://user-images.githubusercontent.com/3756245/82231995-9c00e280-98f3-11ea-985a-b915a0071ee0.png)

![image](https://user-images.githubusercontent.com/3756245/82232026-a58a4a80-98f3-11ea-9808-83e1f90512ec.png)

In the interest of time/workload, I'm going to leave this as is. Feel free to try your suggested restructure in this PR.

Reply:
You're right - after testing it out, I don't think our current tabs implementation can be nested into steps. 

A potential solution would be to break this into its own section - currently, this is all under one "####Configure" heading, so maybe break it right before the tabs into "####Link to the Function", then finally "####Verify" (or something like that).

Reply:
Stet for now.